src/listeners/interactionCreate.ts

```diff
@@ -136,8 +137,16 @@ client.on("interactionCreate", async interaction => {
 								console.error(`Couldn't get string ${path} in English for ${file}, please fix this`)
 						}
 					}
-					if (typeof string === "string" && variables)
+					if (typeof string === "string" && variables) {
 						for (const [variable, text] of Object.entries(variables)) string = string.replace(`%%${variable}%%`, String(text))
+						const locale = lang.replace("_", "-")
```

Comment:
You can probably call formatNumberToLocaleString here instead of calling it on every single instance where a number is passed as a variable